rero_ils/modules/acq_accounts/api.py

```diff
@@ -408,65 +390,78 @@ def get_ancestors(self):
             return [parent] + parent.get_ancestors()
         return []
 
+    def _list_object_by_pid(self, record_class, query):
```

Comment:
This method could be move to parent class `IlsRecord` (rero_ils.modules.api:IlsRecord) and be available for all ReroILS resources.